pkg/prometheus/prometheus.go

```diff
@@ -2,44 +2,26 @@ package prometheus
 
 import (
 	"context"
-	"fmt"
+	"time"
+
 	"github.com/prometheus/client_golang/api"
 	"github.com/prometheus/client_golang/api/prometheus/v1"
-	"time"
+	"github.com/readytalk/stim/pkg/stimlog"
 )
 
 type Prometheus struct {
 	client  *api.Client
 	config  *Config
 	context context.Context
 	API     v1.API
+	log     *stimlog.StimLogger
 }
 
 type Config struct {
 	Address string
-	Logger
-}
-
-type Logger interface {
```

Comment:
I'm not sure I agree with removing the interface and forcing people using this package to use "github.com/readytalk/stim/pkg/stimlog".  The whole idea was to allow a consumer of this package to pass in any logger they wanted as long as it has the Debug and Info methods.

Reply:
I dont completely disagree, but its a little silly to require Stim, which has its logger, then not use its logger.  Right now because of that there is no way these packages can be independent from stim.    

Actually, let me put out the changes I have around redoing stimpacks to make them independent from stim, just didnt want to do it all in one shot.